lib/teiserver_web/templates/admin/autohost/show.html.heex

```diff
@@ -0,0 +1,65 @@
+<div class="container">
+  <h1>Autohost: <%= @autohost.name %></h1>
+
+  <div :if={@client_secret} class="note note-danger">
+    Secret only shown once! <pre><%= @client_secret %></pre>
+  </div>
+
+  <p>
+    <Phx.link href={~p"/teiserver/admin/autohost/#{@autohost.id}/edit"}>
+      <button type="button" class="btn btn-primary">Edit autohost</button>
+    </Phx.link>
+
+    <%!-- TODO: add a modal confirmation for deleting an autohost --%>
```

Comment:
I'm going to leave this out of this PR.
I had a quick look and this may require some upgrade to the bootstrap material design library. This is going to be mostly assets and frontend related work and I'd rather keep the focus of this PR on the simple operations required to manage the oauth bits.